tests/conftest.py

```diff
@@ -33,3 +40,23 @@ def empty_func(*args, **kwargs):
             pass
         _checker = empty_func
     return _checker
+
+
+@pytest.fixture(scope='function')
+def target_opsets(request):
+    opsets = request.config.getoption('opset-versions')
+    min_version = onnx_chainer.MINIMUM_OPSET_VERSION
+    max_version = onnx.defs.onnx_opset_version()
+    if opsets is None:
+        return list(range(min_version, max_version + 1))
+    elif opsets == 'min':
+        return [min_version]
+    elif opsets == 'latest':
+        return [max_version]
+    else:
+        try:
+            versions = [int(i) for i in opsets.split(',') if not i == '']
```

Comment:
I think `if not i == ''` is unnecessary. This allows users to specify "9,,10", but I doubt it's very useful

Reply:
I didn't consider `"9,,10"`, wanted to catch `"9,"`, but I agree your suspicion.